ident/types.go

```diff
@@ -143,6 +143,28 @@ func (ids IDs) Finalize() {
 // Tags is a collection of Tag instances.
 type Tags []Tag
 
+// Finalize finalizes all Tags.
+func (tags Tags) Finalize() {
+	for _, t := range tags {
+		t.Name.Finalize()
+		t.Value.Finalize()
+	}
+}
+
+// Equal returns a bool indicating if the tags are equal.
```

Comment:
nit: perhaps it's worth calling out that the tags must be in the same order?

Reply:
sure thing